cipher/diffiehellman/diffiehellmankeyexchange_test.go

```diff
@@ -39,3 +39,21 @@ func TestDiffieHellmanKeyExchange(t *testing.T) {
 		}
 	})
 }
+
+func FuzzGenSharedKey(f *testing.F) {
+	f.Add(int64(13))
+	f.Fuzz(func(t *testing.T, prvKey int64) {
+		// explicitly ignoring the return error
+		// here we want to test that no strange behaviors are raised when executing GenerateShareKey
+		_ = GenerateShareKey(prvKey)
+	})
+}
+
+func FuzzGenMutualKey(f *testing.F) {
+	f.Add(int64(5), int64(17))
+	f.Fuzz(func(t *testing.T, prvKey, shrdKey int64) {
+		// explicitly ignoring the return error
+		// here we want to test that no strange behaviors are raised when executing GenerateMutualKey
```

Comment:
Can you define "strange" behaviour?

I'm of the opinion that these fuzz tests are for checking the unexpected behaviour - that could only be detected once you check the errors (since there are no `panic`s in the algorithm.)

Reply:
About the reason for these tests: it is mostly to run the function and see if it breaks for unexpected reasons like if there are any issue during the computation done in the algorithm, i.e. divisions by zero, overflows and so on. Let me know what are your thoughts on this.

Reply:
I don't think there is anything that is being checked for us to conclude that there is nothing "strange" happening. I don't think this is a change with any merit... 😓